src/components/create/Create.js

```diff
@@ -0,0 +1,72 @@
+import './Create.css';
+import React, { useState } from 'react';
+
+const Create = ({addDog}) => {
+  const [name, setName] = useState('');
+  const [age, setAge] = useState('');
+  const [breed, setBreed] = useState('');
+  //Will eventually use api that backend creates to turn this into a drop down 
+  const [error, setError] = useState('')
+
+  const submitDog = event => {
+    event.preventDefault();
+    clearInputs()
```

Comment:
I would move this to after the conditional... like line 21.5. better UX in my opinion. Thoughts?